### PR TITLE
Fix: prevents swap when transfer is ongoing

### PR DESCRIPTION
**Pull request by Victor-Salomon**
This PR prevent chains swap when a transfer is ongoing and updates CSS accordingly.

### CODE REVIEW

File: app/src/components/Transfer.tsx [hunk 1]
```diff
@@ -150,7 +150,11 @@ const Transfer: FC = () => {
         />
 
         {/* Switch source and destination chains */}
-        <SwapChains handleChainChange={handleSwapChains} disabled={!allowSwap()} />
+        <SwapChains
+          handleChainChange={handleSwapChains}
+          disabled={!allowSwap()}
+          isTransferOngoing={transferStatus !== 'Idle'}
+        />
```
Comment (noahjoeris) [inline app/src/components/Transfer.tsx:157]:
```suggestion
        <SwapChains
          handleChainChange={handleSwapChains}
          disabled={!allowSwap() || transferStatus !== 'Idle'}
        />
```

**Comment by Victor-Salomon** [comment]
I already included the check within the allowSwap, and the handleSwap. The fact is, the swap can be disabled because the chain swap is not allowed (which relies on the allowSwap) or because of an ongoing transfer (which relies on the status itself). I will check again if I can improve.

**Comment by Victor-Salomon** [comment]
@noahjoeris I can't bypass the cursor modification without taking into consideration an ongoing transfer (which differs from the 'disabled status'). When the component is disabled because of an ongoing transfer, the cursor changes from default to not allowed on hover which is wired. 

Either we: 
- keep the hover cursor switch (I don't like this)
- remove the cursor not allowed to default when the component is disabled (no matter if this is because the chains are not swappable or an ongoing transfer)
- validate the PR with this third props

I would go for option 2

**Comment by noahjoeris** [comment]
Alright thanks. You can decide. I'd also go with option 2 or 3

**Review by noahjoeris — APPROVED** [review]

**Pull request merged**